src/docs/markdown/running.md

```diff
@@ -7,15 +7,19 @@ title: Keep Caddy Running
 While Caddy can be run directly with its [command line interface](/docs/command-line), there are numerous advantages to using a service manager to keep it running, such as ensuring it starts automatically when the system reboots and to capture stdout/stderr logs.
 
 
-- [Linux Service](#linux-service)
-  - [Unit Files](#unit-files)
-  - [Manual Installation](#manual-installation)
-  - [Using the Service](#using-the-service)
-  - [Overrides](#overrides)
-- [Windows Service](#windows-service)
-- [Docker Compose](#docker-compose)
-  - [Setup](#setup)
-  - [Usage](#usage)
+- [Keep Caddy Running](#keep-caddy-running)
```

Comment:
I don't think we need this extra level in the TOC, it's kinda just noise.

Reply:
Arf, my bad it's vscode's Markdown All in One that automatically generates a Table of content on save..